airbyte-integrations/connectors/source-recurly/source_recurly/spec.json

```
@@ -0,0 +1,16 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/recurly",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Recurly Source Spec",
+    "type": "object",
+    "required": ["api_key"],
+    "additionalProperties": false,
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "description": "Recurly API Key, use <a href=\"https://airbyte.recurly.com/integrations/api_keys\">admin</a> to generate this key."
```

current link points them to the Airbyte instance of recurly. Better to redirect to docs instead. 
```suggestion
        "description": "Recurly API Key. See the  <a href=\"https://docs.airbyte.io/integrations/sources/recurly\">docs</a> for more information on how to generate this key."
```